tc_timeval.c

```diff
@@ -26,11 +26,7 @@ SOFTWARE.
 
 int tc_timeval_add(tc_timeval *ret, const tc_timeval *arg1, const tc_timeval *arg2)
 {
-#if PHP_MAJOR_VERSION >= 7
 	zend_long sec, usec;
-#else
-	long sec, usec;
-#endif
```

Comment:
`zend_long` always means 64bit integer on 64bit system,  however `long` has 32bit length on 64bit-Windows system. This fix causes backward-compatibility problem.

Reply:
I've moved this conversion to https://github.com/hnw/php-timecop/pull/31/files/6eb1b043bd863c1b9994c462e962723d2782b359#diff-027212d02ddba994efdbf5792da4389cR33.